third_party/move/move-model/src/builder/module_builder.rs

```diff
@@ -2999,15 +3001,7 @@ impl<'env, 'translator> ModuleBuilder<'env, 'translator> {
             let pool = et.symbol_pool();
             if let Some(entry) = vars.get(name) {
                 // Name already exists in inclusion context, check its type.
-                et.check_type(
-                    loc,
-                    &ty,
-                    &entry.type_,
-                    &format!(
-                        "for `{}` included from schema",
-                        name.display(et.symbol_pool())
-                    ),
-                );
+                et.check_type(loc, &ty, &entry.type_, ErrorMessageContext::General);
```

Comment:
You've lost some functionality here, at least add a TODO so you know where to insert it in a later PR.

It may be easier to add it now: a new enum value `ErrorMessageContext::InSchema(et)` which can include the appropriate message.